common/src/test/java/gov/cms/ab2d/common/SpringBootApp.java

```diff
@@ -0,0 +1,16 @@
+package gov.cms.ab2d.common;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+
+@SpringBootApplication(scanBasePackages = "gov.cms.ab2d.common")
```

Comment:
Can you try just the `SpringBootApplication` annotation _and_ without `scanBasePackages`? package names now match, so auto-config should work.

Reply:
Yeah the auto config works now